core/src/toga/command.py

```diff
@@ -203,7 +203,7 @@ def __init__(
         self.section = section
         self.order = order
 
-        self.action = wrapped_handler(self, action)
+        self._action = wrapped_handler(self, action)
```

Comment:
If we're got a setter, we might as well use it, and avoid duplicating the logic around wrapping.
```suggestion
        self.action = action
```